### PR TITLE
Added booking summary notification for administrators

### PR DESCRIPTION
**Pull request by mingazhev**

### CODE REVIEW

File: Api/Controllers/AgentControllers/InternalBookingsController.cs [hunk 1]
```diff
@@ -158,17 +161,36 @@ public async Task<IActionResult> NotifyPaymentsNeeded(List<int> bookingIds)
...
         {
             return Ok(await _bookingsProcessingService.SendBookingSummaryReports());
         }
+        
+        
+        /// <summary>
+        ///     Sends bookings summary report for administrator
+        /// </summary>
+        /// <returns>Result message</returns>
+        [HttpPost("notify/booking-summary-administrators")]
```
Comment (kirillta) [inline Api/Controllers/AgentControllers/InternalBookingsController.cs:177]: This route is super strange

File: Api/Services/Mailing/BookingMailingService.cs [hunk 2]
```diff
@@ -324,6 +309,53 @@ async Task<Result<string>> SendMails(List<(BookingSummaryNotificationData Data,
...
+                    select new BookingAdministratorSummaryNotificationData.BookingRowData()
+                    {
+                        Agency = agency.Name,
+                        Agent = $"{agent.FirstName} {agent.LastName}",
+                        ReferenceCode = booking.ReferenceCode,
+                        Accommodation = booking.AccommodationName,
+                        Location = $"{booking.Location.Country}, {booking.Location.Locality}",
+                        LeadingPassenger = GetLeadingPassengerFormattedName(booking),
+                        Amount = PaymentAmountFormatter.ToCurrencyString(booking.TotalPrice, booking.Currency),
+                        DeadlineDate = FormatDate(booking.DeadlineDate),
```
Comment (kirillta) [inline Api/Services/Mailing/BookingMailingService.cs:341]: Use MailSender formatters

**Review by kirillta — CHANGES_REQUESTED** [review]

File: Api/Services/Mailing/BookingMailingService.cs (hunk 2 above)
Comment (mingazhev) [reply Api/Services/Mailing/BookingMailingService.cs:341]: Which of them? If you mean EmailContentFormatter.FromDateTime - it is not valid there

File: Api/Controllers/AgentControllers/InternalBookingsController.cs (hunk 1 above)
Comment (mingazhev) [reply Api/Controllers/AgentControllers/InternalBookingsController.cs:177]: Which would you preder?

File: Api/Services/Mailing/BookingMailingService.cs (hunk 2 above)
Comment (kirillta) [reply Api/Services/Mailing/BookingMailingService.cs:341]: Then make a valid one

File: Api/Controllers/AgentControllers/InternalBookingsController.cs (hunk 1 above)
Comment (kirillta) [reply Api/Controllers/AgentControllers/InternalBookingsController.cs]: `administrator-booking-summary` and `agent-booking-summary` I guess

Comment (kirillta) [reply Api/Controllers/AgentControllers/InternalBookingsController.cs]: Not a great name, but it represents a resource at least

**Review by kirillta — APPROVED** [review]

**Pull request merged**